schedule/yast/maintenance/create_hdd_maintenance_migration.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: create_hdd_maintenance_migration
+description: '|
+
+  This job is to deregister the system and publish a qcow
+  file for MU migration test cases.'
```

Comment:
Please, don't use MU, it is not a well know abbreviation, 'maintenance updates'.

Reply:
changed.